src_c/event.c

```diff
@@ -590,6 +620,13 @@ pgEvent_AutoInit(PyObject *self, PyObject *_null)
     if (!_pg_event_is_init) {
         pg_key_repeat_delay = 0;
         pg_key_repeat_interval = 0;
+        if (!pg_evfilter_mutex) {
+            /* Create mutex only if it has not been created already */
+            pg_evfilter_mutex = SDL_CreateMutex();
+            if (!pg_evfilter_mutex)
+                return RAISE(pgExc_SDLError, SDL_GetError());
+        }
+
         SDL_SetEventFilter(pg_event_filter, NULL);
     }
     _pg_event_is_init = 1;
```

Comment:
not sure, what about the `_joy_map_*` methods?

Reply:
Those functions have nothing to do with the event filter, so they don't need mutex safeguards and such